recipes/cigi-cl/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, replace_in_file, rm, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+from conan.tools.apple import is_apple_os
+import os
+
+required_conan_version = ">=1.53.0"
+
+class CigiClConan(ConanFile):
+    name = "cigi-cl"
```

Comment:
```suggestion
    name = "cigi-ccl"
```

I would suggest renaming this project, including the folder to `cigi-ccl`

Repology and Ubuntu are using this name already: https://repology.org/project/ccl-cigi-class-library/versions

There is a "mirror" in github using cigi-ccl as well: https://github.com/Hadron/cigi-ccl

The official web package also mention the ccl: https://cigi.sourceforge.io/product_ccl.php

Reply:
Because `CCL` expands to `Cigi-CL`, calling it `Cigi-CCL` has the word `Cigi` twice in the name.
I don't particularly like this, but I guess it's OK to be consistent with the other packages.

Reply:
>  Better not to use any of that code for reference.

Totally agree.